src/util.js

```diff
@@ -35,8 +35,26 @@ function getMatchingValue(objectLike, stringOrRegex) {
 	}
 }
 
+function getMatchingValuesByAll(arrayLike, arrayOfStringOrRegex) {
+	if (!Array.isArray(arrayOfStringOrRegex)) {
+		return [];
+	}
+
+	const vals = [];
+	for (var i = 0; i < arrayLike.length; i++) {
+		var like = arrayLike[i];
+
+		if (arrayOfStringOrRegex.every((cls) => { return contains(like.class, cls); })) {
```

Comment:
#BecauseIE11, we don't really mention browser support here, but I guess we sorta-kinda-implicitly support it still (via something like My Courses), so can this be a not-array-function?

Reply:
By `not-array-function` are you referring to the `.every`? If so, [MDN shows it having support back to IE9](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every).

Reply:
Whoops, I meant "not-_arrow_-function", my bad :)

Reply:
Oh! I saw ES6 syntax in the library (`const/let`) and just plowed ahead. I'll change it.

Reply:
Oh... huh. I'm... well now I'm just all confused what is and isn't supported in IE11, what is and isn't ES6, and what will and won't work in this library in IE11 😆